src/core/RequestHandler.cpp

```diff
@@ -32,9 +32,25 @@ namespace core {
 		return *this;
 	}
 
-	void RequestHandler::handle(int fd, uint32_t event) {
-		(void)fd;
-		(void)event;
+	// note: the implementation is just temporary
+	void RequestHandler::handle(HandlerContext &ctx) {
+		int fd = ctx.conn.getSocket().getFd();
+		char buffer[1024];
+
+		ssize_t bytesReceived = recv(fd, buffer, sizeof(buffer) - 1, 0);
+		if (bytesReceived < 0) {
+			ctx.conn.close();
+			std::cerr << "Error receiving data" << std::endl;
+			return;
+		}
+		if (bytesReceived == 0) {
+			ctx.conn.close(); //??? why can I close it in read and response? sus
+			return;
```

Comment:
What's so sus? I'd expect you to close the socket here if the client drops the connection, afaik that triggers an event on the fd, which should then spit out 0 bytes.

Reply:
I tried to close this shit everywhere because stuff was not working haha. This code is just temporary anyways to play around

Reply:
Fair. I also have yet to really nail down when what should happen to which connection. But probably best not overthink it.